.angulardoc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "repoId": "4c12df35-5086-47f0-ab90-e3e021deea04",
+  "lastSync": 0
+}
```

Comment:
[remove] - this is a VS Code config file that should not be added to the project.

Reply:
Done